src/BookNavigator/volumes/volumes-provider.js

```diff
@@ -24,40 +24,52 @@ export default class VolumesProvider {
     this.id = 'volumes';
     this.label = `Viewable files (${this.volumeCount})`;
     this.icon = html`${volumesIcon}`;
-    this.actionButton = this.headerIcon;
-    this.sortVolumes(true);
+
+    this.sortOrderBy = "";
+    this.sortVolumes("initial");
+  }
+
+  get sortInitialIcon() {
+    return html`<button class="sort-by neutral-icon" aria-label="Sort volumes in initial order" @click=${() => this.sortVolumes('asc')}>${sortNeutralIcon}</button>`;
   }
 
   get sortAscendingIcon() {
-    return html`<button class="sort-asc icon" aria-label="Sort volumes in ascending order" @click=${() => this.sortVolumes()}>${sortAscendingIcon}</button>`;
+    return html`<button class="sort-by asc-icon" aria-label="Sort volumes in ascending order" @click=${() => this.sortVolumes('desc')}>${sortAscendingIcon}</button>`;
   }
 
   get sortDescendingIcon() {
-    return html`<button class="sort-desc icon" aria-label="Sort volumes in descending order" @click=${() => this.sortVolumes()}>${sortDescendingIcon}</button>`;
+    return html`<button class="sort-by desc-icon" aria-label="Sort volumes in descending order" @click=${() => this.sortVolumes('initial')}>${sortDescendingIcon}</button>`;
   }
 
   get headerIcon() {
-    return this.isSortAscending ? this.sortAscendingIcon : this.sortDescendingIcon;
+    if (this.sortOrderBy === 'initial') return this.sortInitialIcon;
+    else if (this.sortOrderBy === 'asc') return this.sortAscendingIcon;
+    else return this.sortDescendingIcon;
   }
 
-  sortVolumes(initialSort = false) {
-    this.isSortAscending = !this.isSortAscending;
-    const volumesOrderBy = this.isSortAscending ? 'asc' : 'desc';
+  /**
+   * @param {string} sortType (initial, asc, desc)
```

Comment:
Type here as well; You can also add:

```js
/** @typedef {'initial' | 'asc' | 'desc'} SortByType */
```

To the top of the doc, and then reference it by name instead of copying it everywhere. Ie.

```suggestion
   * @param {SortByType} sortType
```

Reply:
Actually, we might want to name these something else; like `'orig_sort' | 'title_asc' | 'title_desc'` to make it a little clearer; took me a long time to realize what each one corresponded to.